app/routes/application.vue

```diff
@@ -0,0 +1,102 @@
+<template lang="html">
+  <div class="">
+<div class="section page">
+<nav class="panel">
```

Comment:
I would likely make this a `form` element so that you can listen for `submit` events.